scripts/rmc.py

```diff
@@ -267,10 +267,11 @@ def get_config(option):
     build_cmd = ["cargo", "build", "--lib", "--target-dir", str(target_dir)]
     if build_target:
         build_cmd += ["--target", str(build_target)]
-    build_env = {"RUSTFLAGS": " ".join(rustflags),
-                 "RUSTC": rustc_path,
-                 "PATH": os.environ["PATH"]
-                 }
+    build_env = os.environ
+    build_env.update({"RUSTFLAGS": " ".join(rustflags),
+                      "RUSTC": rustc_path,
+                      "PATH": os.environ["PATH"]
```

Comment:
Isn't `"PATH"` already `os.environ["PATH"]`?

Reply:
Good catch! I'll remove it.